### PR TITLE
Feature/player inv bot

### PR DESCRIPTION
**Pull request by GDWR**
Add bot side for the API Inventory/Item endpoints created.

closing #21

### CODE REVIEW

File: src/bot/arena_bot.py [hunk 1]
```diff
@@ -1,33 +1,55 @@
...
 import discord
 from discord.ext.commands import Bot
 
+from config import COMMAND_PREFIX
+
 
 class ArenaBot(Bot):
-    def __init__(self, command_prefix, *args, **kwargs):
+    """
+    Subclass of `discord.ext.Bot`
```
Comment (DudeBro249) [inline src/bot/arena_bot.py:15]: `discord.ext.Bot` -> `discord.ext.commands.Bot`

File: src/bot/arena_bot.py [hunk 2]
```diff
@@ -1,33 +1,55 @@
...
+
+    This adds additional functionality:
+     * aiohttp session
+     * Singleton
+     * load_extensions
+    """
+    instance = None
+
+    # flake8: noqa: C901
```
Comment (DudeBro249) [inline src/bot/arena_bot.py:24]: flake8 comment not necessary -> references a PEP standard which is not being raised by flake8 when linting.

File: src/bot/cogs/player.py [hunk 3]
```diff
@@ -8,29 +9,38 @@
 
 
 class Player(Cog):
+    """Cog holds basic `Player` commands such as `join`"""
+
     def __init__(self, bot: ArenaBot):
         self.bot = bot
-        self._api_url = f"http://{API_URL}"
```
Comment (DudeBro249) [inline src/bot/cogs/player.py:13]: Do we even need this property? We can just use the `API_URL` constant directly, right?

**Review by DudeBro249 — CHANGES_REQUESTED** [review]
Great work! Just needs some small changes.

File: src/bot/cogs/player.py [hunk 4]
```diff
@@ -8,29 +9,38 @@
...
+        self._api_url = API_URL
```
Comment (DudeBro249) [inline src/bot/cogs/player.py:16]: Do we even need this property? We can just use the API_URL constant directly, right?

**Comment by GDWR** [comment]
inv == Inventory 
inv != Invite

**Pull request merged**